resources/translations/fi.edn

```diff
@@ -183,6 +185,8 @@
              :show "Näytä"
              :show-less "Näytä vähemmän"
              :show-more "Näytä lisää"}
+  :change-form {:change-form-intro "Olet muuttamassa seuraavien aineistojen lomaketta. Muutokset tehdään yksi kerrallaan siten, että vanha aineisto lopetetaan ja uuteen aktivoitavaan kopioon laitetaan valitsemasi lomake."
```

Comment:
Could it say "poistetaan käytöstä" instead of "lopetetaan"? I think it would be more aligned with the other terminology.